test/squish/suite_nuke_submitter/basic_workflow_gui/test.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# mypy: disable-error-code="attr-defined"
+import names
+import squish_helpers
+import os
+import test
+import squish
+
+
+# This file is a Squish test script for testing the basic workflow of submitting a Nuke job to Deadline Cloud.
+# It automates the process of launching Nuke, opening a script file, configuring AWS profile settings,
+# submitting a job to Deadline Cloud, and then closing Nuke.
+#
+# This script is executed by the Squish test runner when the "basic_workflow_gui" test case is selected.
+# It is triggered using the run_squish_command function in run_squish_test.py in the test_basic_workflow test.
+#
+# The main() function is the entry point that Squish automatically calls when executing this test script.
+
+
+def main():
+    squish_helpers.launch_nuke()
```

Comment:
q: do we need to have any try catch for these? I am not familiar with squish but I am wondering if nuke fails to be launched, will we just fail it or retry the launch?